src/module3/config/index.ts

```diff
@@ -1 +1,36 @@
-export * from './constants';
\ No newline at end of file
+import dotenv from 'dotenv';
+
+// Set the NODE_ENV to 'development' by default
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
+const envFound = dotenv.config();
+if (envFound.error) {
+    // This error should crash whole process
+
+    throw new Error("⚠️  Couldn't find .env file  ⚠️");
+}
```

Comment:
Recommendation: Error scenario can be further improved for specific set of data missing. eg: databaseUrl